spec/fixtures/conf/auth.conf

```diff
@@ -0,0 +1,233 @@
+authorization: {
```

Comment:
The puppet-enterprise-modules PR was merged just recently, though I'm not sure how it gets across to the docker images

Reply:
Based on our conversation during standup, it doesn't. Hence this file, which gets copied to the container.

Reply:
Something minor, but wondering if it would be better to use https://forge.puppet.com/puppetlabs/puppet_authorization to adjust the rule vs coping a auth.conf that is hardcoded - as if new endpoints etc. are exposed we would need to update this auth.conf etc.

Reply:
Then it can be managed with something like:

```
node 'puppet' {
  puppet_authorization::rule { 'puppetlabs v4 catalog for services':
    ensure               => present,
    match_request_path   => '^/puppet/v4/catalog/?$',
    match_request_type   => 'regex',
    match_request_method => 'post',
    allow                => '*',
    sort_order           => 500,
    path                 => '/etc/puppetlabs/puppetserver/conf.d/auth.conf',
  }
}
```

Reply:
Was going to do similar in puppetserver but it wasn't right https://github.com/puppetlabs/puppetserver/pull/2030

Reply:
Yea the above manifest is just to modify the default `deny` to an `allow`. Similar to the PR for puppet-enterprise-modules - but it will be for the docker container. But only downside is, would need to copy the manifest, then apply it - but applying it needs to wait till the container is running which can take a few minutes after starting. Maybe copying the full auth is fine - just need to remember if endpoints are updated/changed in future that it'll need updated.

Reply:
copying in a single file is so much simpler than fiddling around with a module and calling puppet.

for testing purposes this is completely fine.

If anything, we could prune this file down to only allow access to the specific endpoints we're gonna be talking to.